runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/SdkHarnessClient.java

```diff
@@ -300,6 +300,8 @@ public void close() throws Exception {
   private final InstructionRequestHandler fnApiControlClient;
   private final FnDataService fnApiDataService;
 
+  @SuppressWarnings("unchecked") /* SdkHarnessClient does not need to know the type information of
+  BundleProcessor. */
```

Comment:
Can you make this `BundleProcessor<?>` instead?

Reply:
The move is actually for "public <T> BundleProcessor<T> getProcessor.........." but the diff creator picked it as this move.
G3 Linter wants all the overloaded methods to be together.

Reply:
Tried making it BundleProcessor<?> but it did not work.